src/planner.ts

```diff
@@ -743,7 +743,7 @@ export class Planner {
           isDynamicType(command.call.fragment.outputs?.[0]) ||
           (command.call.flags & CommandFlags.TUPLE_RETURN) !== 0
         ) {
-          ret |= 0x80;
+          ret |= 0x00;
```

Comment:
You can delete this line entirely.

Reply:
So that means completely removing the if block?

Reply:
Yes.

Reply:
I think it is important to keep the `if` loop for the dynamic return values like string, array and whatnot so the MSB get set and in the case of tuple we don't need it so we can remove the `or` condition.

So the new change will look like this - 

```
if (isDynamicType(command.call.fragment.outputs?.[0]) ) {
          ret |= 0x80;
}
```